README.md

```diff
@@ -15,18 +15,14 @@ Yoti SDK carries out for you steps 6, 7 ,8 and the profile decryption in step 9.
 
 Yoti also allows you to enable user details verification from your mobile app by means of the Android (TBA) and iOS (TBA) SDKs. In that scenario, your Yoti-enabled mobile app is playing both the role of the browser and the Yoti app. By the way, your back-end doesn't need to handle these cases in a significantly different way. You might just decide to handle the `User-Agent` header in order to provide different responses for web and mobile clients.
 
+## Prerequisites
+Check you have all the following before starting your integration.
+Please ensure you have created your application in our Yoti Application and have downloaded/made a note of all the keys. Keep them safe as you'll need them soon!
 
-## Enabling the SDK
-To import the Yoti SDK inside your project, you can use your favourite dependency management system.
-If you are using NPM, you need to add the following dependency:
+## Supported Versions
+The Yoti Nodejs SDK currently supports all Nodejs versions for Mac users. For Window users please ensure you have version 7.8.0 or later.
```

Comment:
Again, please confirm why you are stating Windows users need 7.8.0 + This SDK has been avaliable before 7.8.0 was even released. You are also ignoring Linux users.